TECHNICAL_VALUES_AND_PRIORITIZATION.md

```diff
@@ -0,0 +1,65 @@
+# Technical Values and Priorities
+
+This document outlines the project's technical values and relative
+prioritization used when making tradeoffs.
+
+## Values and priority level
+
+This is only a subset of the overall priorities, listing only
+those which are at the top of the priority list. Anything on this
+list is very important for the project despite the relative priorities shown.
+
```

Comment:
I propose deleting this text. It seems to water down the actual priorities rather than strengthen them. I don't think we need to be explicit that the list of priorities is not comprehensive.

```suggestion
```

Reply:
The concern was people saying "but what about X" but if other people are not concerned about that I'm ok with removing.

Reply:
I don't think "but what about X?" would be a bad thing to hear. If someone wants to ask "what about X?", then great. Maybe we should add X to the document or explain why we don't include it. My assumption is that this is a living document that gets modified over time, and not something we set in stone.

Reply:
Since nobody else has commented that they'd prefer to keep it will delete.